src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Try to be more descriptive when giving names, because it's not obvious what integer in this map will represent. 
fruitToQuantityMap - one of the possible names